ocs_ci/deployment/hosted_cluster.py

```diff
@@ -1491,3 +1497,50 @@ def odf_csv_installed(self):
             cluster_kubeconfig=self.cluster_kubeconfig,
         )
         return sample.wait_for_func_value(value=True)
+
+    @kubeconfig_exists_decorator
+    def enable_client_console_plugin(self):
+        """
+        Enable the ODF client console plugin by patching the console operator
+
+        Returns:
+            bool: True if the patch is applied successfully, False otherwise
+        """
+        try:
+            self.exec_oc_cmd(
+                "patch console.operator cluster --type json "
```

Comment:
I think this is already implemented in https://github.com/red-hat-storage/ocs-ci/blob/f7ca514dc95b78cd88be98df89674cfa366c5ed0/ocs_ci/ocs/resources/storage_client.py#L130

Reply:
I was thinking to reuse it, but in this case I need redo `enable_console_plugin` because it does not apply with the client kubeconfig. And Provider client deployment does not use multicluster config for deployment but just use modified `self.exec_oc_cmd` to solve issue with setting up new client